kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/BaseConnectorNetworkIT.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright [2020 - 2020] Confluent Inc.
+ */
+
+package io.confluent.connect.s3.integration;
+
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.ListObjectsV2Request;
+import com.amazonaws.services.s3.model.ListObjectsV2Result;
+import com.amazonaws.services.s3.model.S3ObjectSummary;
+import io.confluent.common.utils.IntegrationTest;
+import org.apache.kafka.connect.runtime.AbstractStatus;
+import org.apache.kafka.connect.runtime.rest.entities.ConnectorStateInfo;
+import org.apache.kafka.connect.util.clusters.EmbeddedConnectCluster;
+import org.apache.kafka.test.TestUtils;
+import org.junit.experimental.categories.Category;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+@Category(IntegrationTest.class)
+public abstract class BaseConnectorNetworkIT {
```

Comment:
By extending the existing `BaseConnectorIT` in the later test classes, we can deduplicate and remove most of the code here and leave the networking only relevant functionality.